### PR TITLE
don't create bcolz index by default.

### PR DESCRIPTION
**Pull request by brentp**

### CODE REVIEW

**Comment by bgruening** [comment]
@brentp is there any reason to not create them by default?

**Comment by brentp** [comment]
@bgruening the reason is that we found that most people are doing exomes where queries with any kind of specificity (e.g. impact != "LOW") are already quite fast, especially with the speed improvements to the genotype filters.

we think that in the long run the bcolz indexes (and the general framework that allows them to work) will be great for whole-genome stuff.

that said, it's trivial to create the indicies so we don't think it will affect too many users?

Does this sound reasonable to you?

**Comment by bgruening** [comment]
Yes, this sounds good. Was just wondering what I should enable in Galaxy. I guess bcolz does not have much disadvantage on HPC environments, so I will probably activate it.

File: docs/content/history.rst [hunk 1]
```diff
@@ -16,6 +16,7 @@ Release History
 #. extra VEP annotations are loaded with loading machinery, not as an extra step as before.
 #. add max_aaf_all column (https://github.com/arq5x/gemini/issues/520) as an aggregate of a number of population filters
 #. use --families to limit queries *before* any work is done. Thanks to Bianca for reporting.
+#. no longer create bcolz indicies by default. Users can create them with `gemini bcolz_index`
```
Comment (bgruening) [inline docs/content/history.rst]: missing `.` at the end.

**Comment by bgruening** [comment]
:+1:

**Pull request merged**